gcore/src/utils.rs

```diff
@@ -16,6 +16,23 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
+use gprimitives::{ActorId, CodeId, MessageId, ReservationId};
+
+pub trait AsRawPtr: AsRef<[u8]> + AsMut<[u8]> {
```

Comment:
Sealed as well

Reply:
Currently these are primitive extensions that add methods `as_ptr()`, `as_mut_ptr()`, which are only available inside `gcore` since `gcore::utils` is private. I think since `gcore` already solves problem of moving from raw pointers to rust types and therefore there is no point in making it public and `: Sealed`.

Reply:
I'll mark these extensions as `pub(crate)` for better readability

Reply:
Adjusting vis is even better